molgenis-data-annotators/src/main/java/org/molgenis/data/annotation/cmd/CommandLineAnnotatorConfig.java

```diff
@@ -12,11 +12,13 @@
 import org.molgenis.data.annotation.AnnotationService;
 import org.molgenis.data.annotation.RepositoryAnnotator;
 import org.molgenis.data.annotation.entity.AnnotatorInfo;
+import org.molgenis.data.annotation.utils.JarRunnerImpl;
 import org.molgenis.data.convert.DateToStringConverter;
 import org.molgenis.data.convert.StringToDateConverter;
 import org.molgenis.data.support.AnnotationServiceImpl;
```

Comment:
I have some comments to the AnnotationServiceImpl.
the getAnnotators method is not immutable.  Can this be changed? If no it should be immutable. 
For initialization you can use {}